deploy/deploy.go

```diff
@@ -72,6 +72,27 @@ func logCommand(cmd string, args ...string) error {
 	return c.Run()
 }
 
+// logCommandWithInput runs the specified command but records standard output
+// with log.Info and standard error with log.Warning. in is sent to
+// the standard input of the command.
+func logCommandWithInput(in []byte, cmd string, args ...string) error {
```

Comment:
rewrite logCommand in terms of logCommandWithInput:

```
func logCommand(cmd string, args ...string) error {
    return logCommandWithInput(nil, cmd, args...)
}
```

Reply:
good idea